Dockerfile

```diff
@@ -4,6 +4,10 @@ ARG NOVNC_VERSION="1.4.0"
 ARG OPENWRT_VERSION="23.05.3"
 ARG VERSION_ARG="0.1"
 
+RUN mkdir -p /tmp/noVNC-${NOVNC_VERSION}
+ADD https://github.com/novnc/noVNC/archive/refs/tags/v${NOVNC_VERSION}.tar.gz /tmp/novnc.tar.gz
+ADD https://github.com/bugy/script-server/releases/download/1.18.0/script-server.zip /tmp/noVNC-${NOVNC_VERSION}/script-server.zip
```

Comment:
It doesn't make sense to put script-server.zip into the noVNC folder. Can you change that, please?

In the original code it doesn't make sense either.